include/internal/constant_time.h

```diff
@@ -13,7 +13,7 @@
 
 # include <stdlib.h>
 # include <string.h>
-# include <openssl/e_os2.h>              /* For 'ossl_inline' */
+# include <openssl/e_os2.h>              /* For 'inline' */
```

Comment:
The comment no longer makes sense and it should go. Is the include still needed?

Reply:
It's needed for the `int64_t` types.  I could fix the comment and do anotehr CI pass after you reply to the other comment below.  (And BTW, do you folks assume int64_t exists?  OR do you use ``long long int` ?)

Reply:
Yes, we assume it exists and I can't find any compat header in our portable that would provide it. I think it's because POSIX requires them if you have a 64-bit long long.

Reply:
Added a fixup commit to fix the comment.  What's another CI cycle :)